virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -0,0 +1,23 @@
+[id="virt-cloning-a-datavolume-using-smart-cloning"]
+= Cloning a DataVolume using smart-cloning
+include::modules/virt-document-attributes.adoc[]
+:context: virt-cloning-a-datavolume-using-smart-cloning
+toc::[]
+
+When you clone a DataVolume (DV), you make a copy of it.
+By using cloning, you reduce the amount of time it takes to deploy many
+virtual machines at one time.
+
+Smart-cloning creates a clone of a DV using the most efficient method available,
+either by snapshot or host-assisted cloning.
+
+Smart cloning happens automatically when you create a DV with a
+PersistentVolumeClaim (PVC) source.
+
+include::modules/virt-understanding-smart-cloning.adoc[leveloffset=+1]
+
+include::modules/virt-cloning-a-datavolume.adoc[leveloffset=+1]
+
+== Additional Resources
+
+xref:../../../virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.html#virt-cloning-pvc-of-vm-disk-into-new-datavolume_virt-cloning-vm-disk-into-new-datavolume[Cloning the PersistentVolumeClaim of a virtual machine disk into a new DataVolume]
```

Comment:
Should be a bullet, even if there's only one.

Reply:
Added bullet.